example/hierarchical-jobs/jobs.yaml

```diff
@@ -0,0 +1,145 @@
+apiVersion: batch.volcano.sh/v1alpha1
+kind: Job
+metadata:
+  name: eng-prod-job
+  annotations:
+    "scheduling.k8s.io/group-hierarchy": "root/eng/prod"
```

Comment:
it's better to update `JobSpec` directly for this feature.

and let's use `volcano.sh` instead of `k8s.io` :)